KinSDK/KinSDK/BulkPayments/BatchPaymentTransaction.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  BatchPaymentTransaction.swift
+//  KinSDK
+//
+//  Created by Corey Werner on 14/08/2019.
+//  Copyright © 2019 Kin Foundation. All rights reserved.
+//
+
+import Foundation
+
+public class BatchPaymentTransaction: BaseTransaction {
```

Comment:
aren't you renaming `BaseTransaction` to `Transaction`? or it's required some work because there's a public one already?

Reply:
it requires some additional work. in my queue branch i have made a todo comment and will get back to it